feat(content): Successors: Ghosts I (#12221)
endless-sky/endless-sky: data/successors/predecessors.txt

Reviewer: In case you didn't know: this will add _another_ 10 velocity when the "kis short range" weapon turns into this submunition, speeding up the projectile mid-flight.

		"low shield permeability" 1
		# Their permeable shields won't stop special damage, so this makes these ships less vulnerable to it.
		"heat protection" 1
		"ion protection" 1
		"scramble protection" 1
		"slowing protection" 1
		"corrosion protection" 1
		"discharge protection" 1
		"burn protection" 1
		# For protection against missiles
		"radar jamming" 80
		"optical jamming" 80
		# Engines
		"thrust" 49
		"thrusting shields" 0.84
		"thrusting heat" 1.9
		"flare sprite" "effect/predecessor flare"
			"frame rate" 3
			"rewind"
		"flare sound" "ion large"

		"turn" 1110
		"turning shields" 0.55
		"turning energy" 1.9
		"turning heat" 1.5
		"steering flare sound" "ion large"

		"afterburner thrust" 74
		"afterburner shields" 7.3
		"afterburner heat" 5
		"afterburner energy" 2.9
		"afterburner effect" "hasur afterburner"
		weapon
			"blast radius" 100
			"shield damage" 1000
			"hull damage" 500
			"hit force" 1500
	outfits
		`"Kis," Lethal Wish` 4

		`Hyperdrive`
		`"Jiita" Heirloom Shielding`
		`Successor Cooling` 2

		`"Ija" Heirloom Reactor` 2
		`Heirloom Power Reserve`

		`Multimodal Armor: Regen`
	gun -12 -64
	gun -7 -60
	gun 7 -60
	gun 12 -64
	engine 0 0
		zoom .8

> 	description `One would think that, immured deep within those hostile, rotting abcesses of the stars, the constructs of that foolish race would prove themselves inadequate; that their artificial minds would splinter and falter and crumble like their so-false flesh, festering on the bone. And yet they remain, eternally dying but never dead, consumed by programmed rage and unknowable perception. The constructs will keep their promises, interloper:`

Reviewer 2: "that foolish race" feels like I should know exactly what race it is you're talking about. Maybe even something like

```suggestion
	description `One would think that, immured deep within those hostile, rotting abcesses of the stars, the constructs of a foolish race would prove themselves inadequate; that their artificial minds would splinter and falter and crumble like their so-false flesh, festering on the bone. And yet they remain, eternally dying but never dead, consumed by programmed rage and unknowable perception. The constructs will keep their promises, interloper:`
```

would be clearer?

> 	description `One would think that, immured deep within those hostile, rotting abcesses of the stars, the constructs of the foolish Myiara would prove themselves inadequate; that their artificial minds would splinter and falter and crumble like their so-false flesh, festering on the bone. And yet they remain, eternally dying but never dead, consumed by programmed rage and unknowable perception. The constructs will keep their promises, interloper:`

Reviewer 3: Typo.
```suggestion
	description `One would think that, immured deep within those hostile, rotting abscesses of the stars, the constructs of the foolish Myiara would prove themselves inadequate; that their artificial minds would splinter and falter and crumble like their so-false flesh, festering on the bone. And yet they remain, eternally dying but never dead, consumed by programmed rage and unknowable perception. The constructs will keep their promises, interloper:`
```

		`"Jiita" Heirloom Shielding`
		`Visions of Rime`
		`Successor Cooling`

		`"Ija" Heirloom Reactor` 2
		`Heirloom Power Reserve`

		`Multimodal Armor: Regen`
	gun 0 -70
		under
	turret 32 -21
	turret -32 -21
	engine 0 0
		zoom .5
	description `Queru came last, and was the most inspired of the three, eviscerating at a distance before it could even be seen. For their part, Resja hated it - but by then, Resja's fate had been long sealed.`

effect "queru afterburner"
	sprite "effect/queru afterburner"
		"frame rate" 60
		"no repeat"
	sound "successor afterburner tiny"
	"sound category" "afterburner"
	lifetime 6

# Solitude Enforcer Outfits

outfit `"Iasa," Lethal Word`
	category "Guns"
	series "Guns"
	index 11050
	cost 4200000
	thumbnail "outfit/bimodal coilgun"
	"mass" 17
	"outfit space" -17
	"weapon capacity" -17
	"gun ports" -1
	unique 1
	weapon
		sprite "projectile/iasa"
			"frame time" 3
			"no repeat"
		sound "bimodal"
		"hit effect" "bimodal impact"
		"fire effect" "iasa fire"
		"inaccuracy" 3
		"velocity" 30
		"lifetime" 16
		"reload" 9
		"firing energy" 15
		"firing heat" 21
		"shield damage" 36.6
		"hull damage" 53.4
		"hit force" 18
		"piercing" 0.33
		"firing shields" 11

> 	description `Resja's chosen implement of death, Iasa was a drone ship's perfect coilgun, a fearsome, hungry weapon without respect for the shields of either friends or foes. It remains remarkably efficient, even after fifteen hundred years of waiting.`

Reviewer 3: Also not sure about this one.
```suggestion
	description `Resja's chosen implement of death, Iasa was a drone ship's perfect coilgun, a fearsome, hungry weapon with respect for the shields of neither friend nor foe. It remains remarkably efficient, even after fifteen hundred years of waiting.`
```

			"no repeat"
		"hit effect" "small explosion"
		"hit effect" "kis impact" 3
		"range override" 180
		"inaccuracy" 3
		"velocity" 10
		"lifetime" 20
		"piercing" 0.5
		"inaccuracy" 6
		"submunition" "kis long range" 3

outfit "kis long range"
	weapon
		sprite "projectile/drag cannon"
			"start frame" 1
			"frame rate" 20
			"no repeat"
		"hit effect" "kis impact"
		"velocity" 10
		"lifetime" 100
		"random lifetime" 30
		"shield damage" 43
		"hull damage" 58

effect "kis impact"
	sprite "effect/kis impact"
		"frame rate" 10
		"no repeat"
	lifetime 20
	"velocity scale" 0
	"random velocity" 1

outfit `"Torji," Lethal Sight`
	category "Guns"
	series "Guns"
	index 11060
	cost 7500000
	thumbnail "outfit/bimodal coilgun"
	"mass" 106
	"outfit space" -106
	"weapon capacity" -106
	"gun ports" -1
	weapon
		sprite "projectile/sjeja kinetic lance"
		sound "sjeja lance"
		"fire effect" "moonbeam fleck" 2
		"hit effect" "dragonflame debris"
		"hit effect" "dragonflame plasma"
		"inaccuracy" 3
		"velocity" 40
		"lifetime" 45
		"reload" 24
		"firing energy" 520
		"firing heat" 1360
		"firing shields" 310
		"firing force" 400
		"shield damage" 520
		"hull damage" 720
		"hit force" 400

> 	description `Resja designed a complex loading system to enable this weapon to fire as quickly as it does. The drone ship Queru was built around it, a paired ship and weapon to surpass at range all prior designs of the Myiara.`

Author: Resja is a background character; a long-dead Predecessor engineer who is responsible for designing the drone ships. The player gets a log entry about them in `Successors: Ghosts 11` and other little tidbits throughout the story.

outfit `Crimson Halo`
	category "Systems"
	series "Shields"
	index 11050
	cost 8550000
	thumbnail "outfit/unknown"
	"mass" 31
	"outfit space" -31
	"shield generation" 6.6
	"shield energy" 6.6
	"shield protection" -0.5
	"shield connection point" 1

> 	description `Resja's ships didn't need shields for defense - only as fuel for attacking their foes. Thus, their favored shield cortex is an unusually powerful one, emitting a staggering amount of energy that is much less useful for intercepting damage than it is when fed back into a starship's systems.`

Reviewer 3: Minor nitpick, but "foe" is used kind of a lot?

outfit `Visions of Rime`
	category "Systems"
	series "Cooling"
	index 11020
	cost 5000000
	thumbnail "outfit/unknown"
	mass 10
	"outfit space" -10
	cooling 28
	"shield generation" -0.6
	unique 1

> 	description `Radiating excess heat through a starship's shield matrix, this compact, ancient cooling system is strikingly efficient. On Resja's furious drone ships, the shield strength that this saps is all but insignificant, but on lesser craft it may come with a certain cost to endurance.`

Reviewer 3: "Certain" sounds a bit too ominous of a word for some negative shield generation.
```suggestion
	description `Radiating excess heat through a starship's shield matrix, this compact, ancient cooling system is strikingly efficient. On Resja's furious drone ships, the shield strength that this saps is all but insignificant, but on lesser craft it may come with a cost to endurance.`
```

effect "queru afterburner"
	sprite "effect/queru afterburner"
		"frame rate" 60
		"no repeat"
	sound "successor afterburner tiny"
	"sound category" "afterburner"
	lifetime 6

# Solitude Enforcer Outfits

outfit `"Iasa," Lethal Word`
	category "Guns"
	series "Guns"
	index 11050
	cost 4200000
	thumbnail "outfit/bimodal coilgun"
	"mass" 17
	"outfit space" -17
	"weapon capacity" -17
	"gun ports" -1
	unique 1
	weapon
		sprite "projectile/iasa"
			"frame time" 3
			"no repeat"
		sound "bimodal"
		"hit effect" "bimodal impact"
		"fire effect" "iasa fire"
		"inaccuracy" 3
		"velocity" 30
		"lifetime" 16
		"reload" 9
		"firing energy" 15
		"firing heat" 21
		"shield damage" 36.6
		"hull damage" 53.4
		"hit force" 18
		"piercing" 0.33
		"firing shields" 11
	description `Resja's chosen implement of death, Iasa was a drone ship's perfect coilgun, a fearsome, hungry weapon with respect for the shields of neither friend nor foe. It remains remarkably efficient, even after fifteen hundred years of waiting.`

effect "iasa fire"
	sprite "effect/emp torpedo fire"
		"frame time" 3
		"random start frame"
		scale 0.5
	lifetime 3

> outfit `"Kis," Lethal Wish`

Reviewer 4: The current outfits with a non-human name seem to have the name taken in quotes (as is here), but then without the comma `,`. Would it maybe make sense to make it align in style and change to `"Kis" Lethal Wish`? Many outfits below in this PR also use the format `"Name" addition addition`.

This applies to some outfits below as well.
